docs/output.md

```diff
@@ -101,23 +101,57 @@ The files will be phased if the flag `--phase` is used (with `_phased` suffix).
 
 ## Imputation outputs `--steps impute`
 
-The results from steps impute will have the following directory structure:
+The results from `--steps impute` will have the following directory structure:
 
 <details markdown="1">
 <summary>Output files</summary>
 
 - `imputation/csv/`
   - `impute.csv`: A single csv containing the path to a VCF and its index, of each imputed sample with their corresponding tool.
 - `imputation/[glimpse1,glimpse2,quilt,stitch]/`
-  - `concat/*.vcf.gz`: A VCF of each imputed sample.
-  - `concat/*.vcf.gz.tbi`: A tbi for the imputed VCF.
-  - `samples/*.vcf.gz`: A VCF of each imputed sample.
-  - `samples/*.vcf.gz.tbi`: A tbi for the imputed VCF.
+  - `concat/all.batch*.vcf.gz`: The concatenate vcf of all imputed samples by batches.
+  - `concat/all.batch*.vcf.gz.tbi`: A tbi for the concatenated samples imputed vcf.
+  - `samples/*.vcf.gz`: A vcf of each imputed sample.
+  - `samples/*.vcf.gz.tbi`: A tbi for the imputed vcf.
 
 </details>
 
 [`bcftools concat`](https://samtools.github.io/bcftools/bcftools.html#concat) will produce a single VCF from a list of imputed VCFs in chunks.
 
+## Simulation outputs `--steps simulate`
+
+The results from `--steps simulate` will have the following directory structure:
```

Comment:
We could add the tree directory structure like we did with --panelprep

Reply:
Done